src/pkg/control/backup.go

```diff
@@ -0,0 +1,96 @@
+package control
+
+import (
+	"github.com/alcionai/corso/src/pkg/extensions"
+)
+
+// DefaultBackupOptions provides a Backup with the default values set.
+func DefaultBackupOptions() Backup {
+	return Backup{
+		FailureHandling: FailAfterRecovery,
+		DeltaPageSize:   500,
+		Parallelism: Parallelism{
+			CollectionBuffer: 4,
+			ItemFetch:        4,
+		},
+	}
+}
+
+// Backup is the set of options used for backup operations. Each set of options
+// is only applied to the backup operation it's passed to. To use the same set
+// of options for multiple backup operations pass the struct to all operations.
+type Backup struct {
```

Comment:
Nit: `Backup` is a little generic.  Can we do this to match how we've named the restore counterpart?
```suggestion
type BackupConfig struct {
```